<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lblIndex">
      <property name="text">
       <string>序号列：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditIndexX">
      <property name="text">
       <string>A</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="lblIndex_2">
      <property name="text">
       <string>BugID列：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="lblIndex_3">
      <property name="text">
       <string>关键字</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLineEdit" name="lineEditBugIdKeyWord">
      <property name="text">
       <string>BugID</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLabel" name="lblIndex_4">
      <property name="text">
       <string>列号</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QLineEdit" name="lineEditBugIdX">
      <property name="text">
       <string>M</string>
      </property>
     </widget>
    </item>
    <item row="0" column="8">
     <widget class="QLabel" name="lblIndex_5">
      <property name="text">
       <string>Bug等级列：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="9">
     <widget class="QLabel" name="lblIndex_6">
      <property name="text">
       <string>关键字</string>
      </property>
     </widget>
    </item>
    <item row="0" column="10" colspan="2">
     <widget class="QLineEdit" name="lineEditBugLevelKeyWord">
      <property name="text">
       <string>BugLevel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="12">
     <widget class="QLabel" name="lblIndex_7">
      <property name="text">
       <string>列号</string>
      </property>
     </widget>
    </item>
    <item row="0" column="13">
     <widget class="QLineEdit" name="lineEditBugLevelX">
      <property name="text">
       <string>M</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="lblIndex_8">
      <property name="text">
       <string>文件目录：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="6">
     <widget class="QLineEdit" name="lineEditDir">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="8">
     <widget class="QPushButton" name="btnSelectDir">
      <property name="text">
       <string>选择目录</string>
      </property>
     </widget>
    </item>
    <item row="1" column="9" colspan="2">
     <widget class="QPushButton" name="btnStart">
      <property name="text">
       <string>开始提取</string>
      </property>
     </widget>
    </item>
    <item row="1" column="11">
     <widget class="QPushButton" name="btnClearData">
      <property name="text">
       <string>清除</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="14">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>958</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
   <property name="toolTip">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
